front/app/components/UI/MultipleSelect/index.tsx

```diff
@@ -20,85 +20,72 @@ export type Props = {
   isSearchable?: boolean;
 };
 
-interface State {}
-
-export default class MultipleSelect extends React.PureComponent<Props, State> {
-  private emptyArray: never[];
-
-  constructor(props: Props) {
-    super(props);
-    this.emptyArray = [];
-  }
-
-  handleOnChange = (newValue: IOption[]) => {
-    // TODO: Fix this the next time the file is edited.
-    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-    this.props.onChange(newValue || this.emptyArray);
+const MultipleSelect = ({
```

Comment:
Nothing really changed here - just converted it from class to functional component.